app/domain/authentication/authn_oidc/authenticate.rb

```diff
@@ -69,12 +69,9 @@ def add_username_to_input
         @authenticator_input = @authenticator_input.update(username: conjur_username)
       end
 
+      # The credentials are in a URL encoded form data in the request body
```

Comment:
We talked about how credentials may not be too descriptive so what would credentials for us? I would add what we expect here in a comment

Reply:
the credentials differ from authenticator to authenticator. For example, in OIDC the credentials are an ID Token. In authn-azure it is an Azure access token. In authn it is an API Key.